src/common/Types.ts

```diff
@@ -522,6 +522,7 @@ export interface ViewPropsShared extends CommonProps, CommonAccessibilityProps {
     viewLayerTypeAndroid?: ViewLayerType; // Android only property
     children?: ReactNode;
     focusable?: boolean;
+    importantForLayout?: boolean; // Web-only, additional invisible DOM elements will be added to track the size changes faster
```

Comment:
This is a better solution than before, but it's still a bit ugly. I can't think of any better alternative at this point though.

This prop will need to be documented. Please add it to the RX.View document.